example/lib/main.dart

```diff
@@ -27,12 +27,19 @@ class _MyAppState extends State<MyApp> {
   var staticMapProvider = new StaticMapProvider(API_KEY);
   Uri staticMapUri;
 
+  List<Marker> _markers = <Marker>[
+    new Marker("1", "Work", 45.523970, -122.663081, color: Colors.blue),
+    new Marker("2", "Nossa Familia Coffee", 45.528788, -122.684633),
+  ];
+
   @override
   initState() {
     super.initState();
-    cameraPosition = new CameraPosition(Locations.portland, 12.0);
-    staticMapUri = staticMapProvider.getStaticUri(Locations.portland, 12,
-        width: 900, height: 400);
+    cameraPosition = new CameraPosition(Locations.portland, 2.0);
+    staticMapUri = staticMapProvider.getStaticUriWithMarkers(_markers,
+      maptype: StaticMapViewType.satellite, center: new Location(45.523970, -122.663081));
+    //staticMapUri = staticMapProvider.getStaticUri(Locations.portland, 12,
```

Comment:
I'd prefer to leave this line uncommented and remove the line you added for testing the "center" attribute.